backend/src/helpers/organization.ts

```diff
@@ -65,6 +104,227 @@ export const createOrganization = async ({
   return organization;
 };
 
+/**
+ * Delete organization with id [organizationId]
+ * @param {Object} obj
+ * @param {Types.ObjectId} obj.organizationId - id of organization to delete
+ * @returns 
+ */
+export const deleteOrganization = async ({
+  organizationId
+}: {
+  organizationId: Types.ObjectId;
+}) => {
+  const session = await mongoose.startSession();
```

Comment:
To support mongodb transaction. I think the user's need to enable replicaSet in infra level at mongodb. 

Don't you think this would break  self hosted users

Reply:
I was not aware actually; will discuss with @maidul98

Reply:
mongodb >= 3.6.0 supports transactions. i also checked document db and it also supports transactions

Reply:
good use of transactions here

Reply:
Sounds like we're good here then.